amazon/amazon.go

```diff
@@ -32,26 +33,32 @@ var (
 		"2022": "https://cdn.amazonlinux.com/al2022/core/mirrors/latest/x86_64/mirror.list",
 		"2023": "https://cdn.amazonlinux.com/al2023/core/mirrors/latest/x86_64/mirror.list",
 	}
+	extrasListURI = map[string]string{
+		"2": "https://cdn.amazonlinux.com/2/extras-catalog-x86_64.json",
```

Comment:
Can you add a comment where you found this link?
It may help to add links for new releases next time.